src/viam/examples/modules/CMakeLists.txt

```diff
@@ -27,35 +27,21 @@ install(
 )
 
 
-add_executable(example_mlmodel
-  example_mlmodel.cpp
-)
-
-target_link_libraries(example_mlmodel
-  PRIVATE Threads::Threads
-  viam-cpp-sdk::viamsdk
-)
-
-install(
-  TARGETS example_mlmodel
-  COMPONENT examples
-)
-
 find_path(TFLITE_INCLUDE_PATH "tensorflow/lite/c/c_api.h")
 find_library(TFLITE_LIBRARY "tensorflowlite_c")
 if (EXISTS ${TFLITE_INCLUDE_PATH} AND EXISTS ${TFLITE_LIBRARY})
-  add_executable(mlmodelservice_tflite
-    mlmodelservice_tflite.cpp
+  add_executable(example_mlmodelservice_tflite
```

Comment:
I thought it would be a good idea for the binary we produce here to include `example`, unless/until it becomes something more.